components/sup/src/templating/context.rs

```diff
@@ -37,12 +37,16 @@ impl<'a> Binds<'a> {
 
 #[derive(Clone, Debug, Serialize)]
 pub struct BindGroup<'a> {
-    members: Vec<SvcMember<'a>>,
+    pub first: Option<SvcMember<'a>>,
```

Comment:
Optional binds may never populate with members, so we need to have this be an `Option`